doc/manual/getting_started_core.rst

```diff
@@ -176,6 +176,25 @@ Within a parallel block, some statements can be made sequential again using a ``
 
 .. _rtio-analyzer-example:
 
+A for loop inside of a parallel block implicitly shifts to sequential execution of sub-statements, since the for loop is considered the top-level statement.
+If, for convenience, a predefined list of statements needs to be executed in parallel by iterating over the list in a loop, then the timeline cursor must be reset manually.
+An example of this is provided below: ::
+
+    # set the dio channel in a for loop using funciton pointers
+    with parallel:
+        parallel_start_mu = now_mu()
+        for pulse in self.list_of_objects_with_run_method:
+            at_mu(parallel_start_mu)
+            pulse.run()
+        delay(10 * ms)
+
+.. note::
+    Using a for loop to unroll many parallel execution statements can cause the scalable event dispatcher (SED) to throw an RTIO sequence error.
+    This happens if the number of simultaneous events exceeds the number of SED lanes.
```

Comment:
There are other cases where this would happen, see https://github.com/m-labs/artiq/issues/1310